tests/test_execution.py

```diff
@@ -135,6 +137,7 @@ def test_traintuple_execution_failure(factory, client, default_dataset_1):
 
 
 @pytest.mark.slow
+@pytest.mark.remote_only
```

Comment:
I think this should work on local backend as well.

Reply:
The tests on 'failure' do not work with the local backend since in the local backend an error is raised whereas in the remote the status is set to "failed".
I can adapt the test to `with pytest.raises` if it is on the local backend though

Reply:
Ah yes, I get it. Well, using pytest.raises here would be great!